commcare/src/test/java/org/motechproject/commcare/events/CaseActionServiceImplTest.java

```diff
@@ -23,52 +20,32 @@
 import static org.mockito.Matchers.eq;
 import static org.mockito.Mockito.verify;
 
-public class CaseEventHandlerTest {
+public class CaseActionServiceImplTest {
```

Comment:
why do you remove this tests ? You should update old tests like CaseEventHandlerTest to pass after your changes. And write new one to the recently created services. However new tests please add in separate PR